pkg/scheduler/ugm/group_tracker.go

```diff
@@ -92,6 +92,7 @@ func (gt *GroupTracker) clearLimits(queuePath string) {
 	gt.queueTracker.setLimit(strings.Split(queuePath, configs.DOT), nil, 0, false, group, false)
 }
 
+// Note: headroom is not read-only, it also traverses the queue hierarchy and creates childQueueTracker if it does not exist.
```

Comment:
maybe we should say "headroom of queue tracker". it is more precise to me

Reply:
Fixed.